logging/log_collection_forwarding/cluster-logging-collector.adoc

```diff
@@ -11,6 +11,9 @@ toc::[]
 
 You can configure the CPU and memory limits for the log collector and xref:../../logging/config/cluster-logging-moving-nodes.adoc#cluster-logging-moving[move the log collector pods to specific nodes]. All supported modifications to the log collector can be performed though the `spec.collection.log.fluentd` stanza in the `ClusterLogging` custom resource (CR).
 
+include::modules/configuring-logging-collector.adoc[leveloffset=+1]
+// https://github.com/openshift/cluster-logging-operator/blob/release-5.8/docs/reference/operator/api.adoc
```

Comment:
Are we going to no longer pull these into the official docs?

Reply:
We will, but it's not as high priority as finishing the restructuring first, I actually meant to remove this comment because I remembered that we can't link to Github, which I was thinking about doing as a temporary measure.